packages/core/dist/components/MonthPicker/MonthPicker.js

```diff
@@ -142,7 +143,11 @@ var MonthPicker = exports.MonthPicker = function (_React$PureComponent) {
         var selectedMonths = monthNumbers.filter(function (m) {
           return !disabledMonths.includes(m);
         });
-        this.setState({ selectedMonths: selectedMonths });
+        this.setState({
+          selectedMonths: selectedMonths,
+          selectAllPressed: true,
+          clearAllPressed: false
```

Comment:
Forgot to revert these

Reply:
Not sure why it isn't showing the latest version I committed last night. The file is correct in Sawyer's comment below: https://github.com/CMSgov/design-system/pull/235/files/1433a682fef50adfa8cc35c69dfafb8c04e048f2..9647e024ca0d412a12b45839bade5d704c08bf06#diff-6c5bf996b9de0d202168b671faa4c0da